src/protocol/logpoint.ts

```diff
@@ -55,75 +55,77 @@ export const PointHandlers: {
   onPoints?: (points: PointDescription[], info: LogpointInfo) => void;
 } = {};
 
-client.Analysis.addAnalysisResultListener(({ analysisId, results }) => {
-  log(`AnalysisResults ${results.length}`);
+export function setupLogpoints() {
```

Comment:
why did this need to be in a setup function?

could we move the handler bits out `({ analysisId, results }) => {})`

so it reads

```js
export function setupLogpoints() {
  client.Analysis.addAnalysisResultListener((onAnalysisResult))
}
```